tests/CoreFX/CoreFX.issues.json

```diff
@@ -696,6 +696,20 @@
             ]
         }
     },
+    {
+        "name": "System.Drawing.Tests",
+        "enabled": true,
+        "exclusions": {
+            "namespaces": null,
+            "classes": null,
+            "methods": [
+                {
+                    "name": "System.Drawing.Tests.FontTests.ToLogFont_DisposedGraphics_ThrowsArgumentException",
```

Comment:
The Unix implementation of Font lets the exception thrown by GCHandle internally escape.